app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,25 @@
+# show each individual question
+get '/questions/:id' do
+  question = Question.find(params[:id]).next
+  survey = TakenSurvey.find(params[:survey_id])
```

Comment:
taken_survey = TakenSurvey.find(params[:survey_id])

###want to make sure we keep the naming conventions the same (survey has to do with surveys that are created, not surveys that are taken :santa: )